src/api/common_api.js

```diff
@@ -811,11 +811,12 @@ module.exports = {
         },
         nsfs_account_config: {
             type: 'object',
-            required: ['uid', 'gid', 'new_buckets_path'],
+            required: ['uid', 'gid', 'new_buckets_path', 'allowed_non_fs_buckets'],
             properties: {
                 uid: { type: 'number' },
                 gid: { type: 'number' },
                 new_buckets_path: { type: 'string' },
+                allowed_non_fs_buckets: { type: 'boolean' }
```

Comment:
How about we rename (and reverse meaning) to `nsfs_only`?

Reply:
renamed, make more sense, thank you.